source/fundamentals/crud/read-operations/specify-query.txt

```diff
@@ -0,0 +1,286 @@
+.. _csharp-specify-query:
+
+===============
+Specify a Query
+===============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+In this guide, you can learn how to specify a query to match a subset
+of documents using the {+driver-long+}.
+
+Most CRUD operations allow you to narrow the set of matched documents
+by passing an expression as an argument to the method, or by specifying
+matching criteria in a **query filter**. Query filters contain one or more query
+operators to determine which documents to include in the resulting set.
+
+.. tip:: Find All Documents
+
+   Passing an empty query filter matches all documents in the collection.
+
+.. tip:: Using LINQ
+
+   You can also specify queries using LINQ. To learn more about using LINQ, see
+   :ref:`csharp-linq`.
+
+The examples in this guide use the following documents in a collection called
+``guitars``.
+
+.. code-block:: json
+ 
+   { "_id": 1, "make": "Fender", "models": ["Stratocaster", "Telecaster"], "establishedYear": 1946, "rating": 9 }
+   { "_id": 2, "make": "Gibson", "models": ["Les Paul", "SG", "Explorer"], "establishedYear": 1902, "rating": 8 }
+   { "_id": 3, "make": "PRS", "models": ["Silver Sky", "SE", "Custom"], "establishedYear": 1985, "rating": 9 }
+   { "_id": 4, "make": "Kiesel", "models": ["Ares", "Vader", "Solo"], "establishedYear": 2015 }
+   { "_id": 5, "make": "Ibanez", "models": ["RG", "AZ"], "establishedYear": 1957, "rating": 7 }
+   { "_id": 6, "make": "Strandberg", "models": ["Boden", "Salen"], "establishedYear": 1982 }
+
+The following ``Guitar`` class models the documents in this collection:
+
+.. literalinclude:: ../../../includes/fundamentals/code-examples/specify-query/Guitar.cs
+   :language: csharp
+   :copyable:
+   :dedent:
+
+Literal Values
+--------------
+
+Literal value queries return documents with an exact match to your search criteria.
```

Comment:
Question/Issue: Is there a reason "search criteria" is used here instead of "query filter"? I think query filter was defined earlier? Should "search criteria" be defined?